treeherder/etl/artifact.py

```diff
@@ -17,7 +17,7 @@ def store_text_log_summary_artifact(job, text_log_summary_artifact):
     """
     errors = json.loads(text_log_summary_artifact['blob'])['errors']
 
-    TextLogError.objects.bulk_create(
+    qs = TextLogError.objects.bulk_create(
```

Comment:
will this be null || 0 if there are no errors, i.e. no return value of {'status': 'OK'} ?